pkg/compactor/bucket_compactor.go

```diff
@@ -279,7 +280,16 @@ func (c *BucketCompactor) runCompactionJob(ctx context.Context, job *Job) (shoul
 	toCompactMaxTime := maxTime(toCompact)
 	jobLogger = log.With(jobLogger, "minTime", toCompactMinTime.String(), "maxTime", toCompactMaxTime.String())
 
-	level.Info(jobLogger).Log("msg", "compaction available and planned; downloading blocks", "blocks", len(toCompact), "plan", fmt.Sprintf("%v", toCompact))
+	var sb strings.Builder
+	for i, meta := range toCompact {
+		if i > 0 {
+			sb.WriteRune(',')
+		}
+		sb.WriteString(meta.ULID.String())
+	}
+	toCompactStr := sb.String()
+
+	level.Info(jobLogger).Log("msg", "compaction available and planned; downloading blocks", "blocks", len(toCompact), "plan", toCompactStr)
```

Comment:
While we're changing this, could we be consistent and log list of blocks for `blocks` key, and move blocks count to say `blocksCount` key? Same on line 343. (Messages on lines 360 and 373 use `blocks` for list of blocks already). WDYT?

Reply:
Makes sense to me @pstibrany, will do.